web/themes/custom/pineapple/pineapple.theme

```diff
@@ -33,6 +33,20 @@ function pineapple_theme_suggestions_field_alter(array &$suggestions, array $var
   }
 }
 
+/**
+ * Implements hook_theme_suggestions_input_alter().
+ */
+function pineapple_theme_suggestions_input_alter(array &$suggestions, array $variables) {
+  $element = $variables['element'];
+  if (!empty($element['#name'])) {
+    $suggestions[] = 'input-' .$element['#name'];
+  }
+  if (!empty($element['#webform_id'])) {
+    $suggestions[] = 'input-' . $element['#webform_id'] . '-' . $element['#name'];
+    $suggestions[] = 'input-' . $element['#webform_id'] . '-' . $element['#type'];
```

Comment:
Type should be a different if statement and you should be checking if both variables used inside the if statement are not empty

Reply:
Done